docs/site/index/survey-tool/managing-users.md

```diff
@@ -14,68 +14,83 @@ Open the survey tool, and click on the gear icon in the top left, then pick **Li
+
+Note that you shouldn't use your TC or Manager account for voting; that should only be done for carrying out committee decisions. What most people do is open a second account as a Vetter if they have a language they want to contribute to.
 
 ## To add a user
 
 Click on \[Add User] at the top:
-![image](../../images/index/managingUsers1.png)
 
-➡️
+![image](../../images/index/managingUsers1.png)
 
 ![image](../../images/index/managingUsers2.png)
 
-Set the name, email, and locales responsible. To see the locale codes, go to http://unicode.org/cldr/apps/survey?p_codes=t. Just use the language codes in on the right side\-\- the country parts are ignored.
-- *Note that you must use the right code in the case of synonyms: use he* ***not*** *iw, use pt* ***not*** *pt\_BR...*
+Set the name, email, and the locale(s) they are responsible for.
 
 ![image](../../images/index/managingUsers3.jpg)
 
 Set the Userlevel based on your assessment of the person's skill. Normally this is "Vetter". You can use "Street" for volunteers whom you don't know. People don't have to come from your organization as long as you have confidence in them.
+
 - See the [CLDR Process](/index/process) page for a description of the user levels and their privileges.
 
-*Known bug: if you try to create a user when the user already has an account, you get an unknown error message. Often this is because the user is locked. Click on "Show locked users", find the person, and change the status (see below).*
+*Known bug: if you try to create a user and that email address already has an account, you get an unknown error message. Often this is because the user is locked. Click on "Show locked users", find the person, and change the status (see below).*
+
+## To forward old votes to a new user
+When replacing a previous user with a new one, you will likely want that new user to inherit the votes from their predecessor. Otherwise the new user would have to submit all votes from scratch manually, which can be several thousands.
+
+1. Once the new user has been created, find them in the list of users and click on the link "Copy votes to this vetter"
+2. Choose the user's predecessor (i.e. the user you want to copy votes *from*).
+3. Choose what locale you want to copy votes *from*. (In case the old user has been active in multiple locales.)
+4. Choose what locale you want to copy the old user's votes *to*. (In almost all cases, that will be the same locale, which is the default choice.)
+5. Vote copying will take place.
+6. Tell the new user to exit and re-enter the survey tool.
+
+At this time, the survey tool will consider the old user's voting history the same as the new user's. When the new user logs in, survey tool will automatically apply all the old user's votes **when they match the currently shipping value**. In any case where the old vote does not match the currently shipping value OR where the value is new, the vote will be abstained and the new user will have to submit a new vote.
```

Comment:
This makes it sounds like I can only import votes into a new user (before they have logged in to the current cycle).

If the vetter has logged in area, can a Manager still import votes and have them stick. (All those that don't conflict with a blue star, right??)

Reply:
they can import votes to a non-new user, but they won't "stick" for xpaths that:
- they have already done an explicit vote on
- they already have an existing import that they've performed
a message comes up saying 'skipping this xpath' or some such

Reply:
We don't intend to cover every case under the moon either. The most common scenario for this feature is when we introduce a new vetter, and we don't say in this text that this is the only scenario this feature could be used, so I think we can stick with the current wording for now.